Microsoft.Toolkit.HighPerformance/Helpers/ParallelHelper.ForEach.IRefAction.cs

```diff
@@ -135,8 +135,7 @@ public static partial class ParallelHelper
             /// Processes the batch of actions at a specified index
             /// </summary>
             /// <param name="i">The index of the batch to process</param>
-            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Expected for the inlining to disappear?

Reply:
Yeah this was just a leftover, it's not there in the other implementations either. This method is just called once per thread anyway and does all the job. It's the actual delegates within each invocation of this method that need to be inlined, as they're invoked in a tight loop, not this one. As in, this loop here:

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/2745962d560c4cf96b55074bc8babd0da7c6b0b3/Microsoft.Toolkit.HighPerformance/Helpers/ParallelHelper.ForEach.IRefAction.cs#L147-L152